edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.CloudProxy/DeviceScopeApiClient.cs

```diff
@@ -133,6 +154,60 @@ async Task<ScopeResult> GetIdentitiesInScope(Uri uri)
             }
         }
 
+        async Task<ScopeResult> GetIdentitiesInTargetScopeWithRetry(Uri uri, Option<string> continuationToken)
+        {
+            try
+            {
+                return await ExecuteWithRetry(
+                    () => this.GetIdentitiesInTargetScope(uri, continuationToken),
+                    Events.RetryingGetIdentities,
+                    this.retryStrategy);
+            }
+            catch (Exception e)
+            {
+                Events.ErrorInScopeResult(e);
+                throw;
+            }
+        }
+
+        async Task<ScopeResult> GetIdentitiesInTargetScope(Uri uri, Option<string> continuationLink)
+        {
+            HttpClient client = this.proxy
+                .Map(p => new HttpClient(new HttpClientHandler { Proxy = p }, disposeHandler: true))
+                .GetOrElse(() => new HttpClient());
+            using (var msg = new HttpRequestMessage(HttpMethod.Post, uri))
+            {
+                if (!this.AuthChainProvider.TryGetAuthChain(this.deviceId, out string authChain))
+                {
+                    throw new ArgumentException($"No valid authentication chain for {this.deviceId}");
+                }
+
+                var payload = new NestedScopeRequest()
+                {
+                    AuthChain = authChain,
+                    PageSize = this.batchSize,
+                    ContinuationLink = continuationLink.GetOrElse(() => string.Empty)
```

Comment:
Rename "ContinuationLink" to "ContinuationToken"?
```suggestion
                    ContinuationToken = continuationToken.GetOrElse(() => string.Empty)
```